doc/flame/inputs/keyboard-input.md

```diff
@@ -91,6 +91,31 @@ that triggered the callback in the first place. The second is a set of the curre
 The returned value should be `true` to allow the continuous propagation of the key event among other
 components. To not allow any other component to receive the event, return `false`.
 
+Flame also provides a default implementation called `KeyboardListenerComponent` which can be used to handle
+keyboard events which can be composed with a `FlameGame` or any other `Component`:
+
+For example, imagining a `PositionComponent` which has methods to move on the X and Y axis,
+the following code could be used to bind those methods to key events:
```

Comment:
```suggestion
Flame also provides a default implementation called `KeyboardListenerComponent` which can be used to handle
keyboard events.  It can be used as a mixin with a `FlameGame` or any other `Component`, or just extended:

For example, imagine a `PositionComponent` which has methods to move on the X and Y axis,
then the following code could be used to bind those methods to key events:
```